sdk/appconfiguration/tests.native.yml

```diff
@@ -19,9 +19,5 @@ extends:
       safeName: azuredataappconfiguration
     TimeoutInMinutes: 90
     SupportedClouds: 'Public'
-    EnvVars:
```

Comment:
With no `UseFederatedAuth: true` these tests will not run with federated auth.

Reply:
added.